src/index.ts

```diff
@@ -48,7 +49,30 @@ const headlessReducerManager = new HeadlessReducerManager();
  *
  * @public
  */
-export function provideAnswersHeadless(config: HeadlessConfig): AnswersHeadless {
+export function provideAnswersHeadless(config: HeadlessConfig): AnswersHeadless;
+
+/**
+ * Supplies a new instance of {@link AnswersHeadless}, using the provided configuration,
```

Comment:
nit: I was thinking we could put the docs above the just above the public implementation to avoid duplication